public/app/screens/main/sections/settings/style.scss

```diff
@@ -0,0 +1,48 @@
+@import "theme";
+
+.cc-settings {
+  padding: 40px;
+
+  .cc-settings__title {
```

Comment:
Prefer $-notation throughout your whole your scss file. Here it should say:

    &__title {